cargo-afl/src/bin/cargo-afl.rs

```diff
@@ -310,7 +310,8 @@ where
          -C target-cpu=native "
     );
 
-    if cfg!(not(feature = "no_cfg_fuzzing")) {
+    let no_cfg_fuzzing = env::var("AFL_NO_CFG_FUZZING").unwrap_or_default();
+    if no_cfg_fuzzing.is_empty() {
```

Comment:
Note from author:

Alternatively I was thinking also of adding else clause with:
- removing AFL_NO_CFG_FUZZING here
- setting config `no_fuzzing`

and not adding env to the `Command` in line 347.

It would be more elegant than this. But that `no_fuzzing` config could be somehow confusing.

Reply:
So, essentially, cargo-afl would turn the environment variable into a config option?

I think I like that idea.

That way, afl/build.rs should not need to consider environment variables at all (right?).

Apologies, I am traveling (hence, the funny account) and the earliest I will be able to merge anything will be Monday evening ET.

Reply:
Also, let's please use just `.is_ok()` to check whether `AFL_NO_CFG_FUZZING` is enabled. That is the convention used for other environment variables, currently.